tools/make/depcheck/src/allowlist.rs

```diff
@@ -173,10 +176,17 @@ pub const EXTRA_SOURCE_DEPS: &[&str] = &[
     "ryu",
     "serde-aux",
     "serde_json",
+    "serde_spanned",
     "static_assertions",
     "tar",
+    "toml_edit",
+    "toml_datetime",
+    "thiserror",
+    "thiserror-impl",
+    "typeid",
     "toml",
     "twox-hash",
+    "winnow",
 ];
```

Comment:
Thought: Lots added and not removed. Intentional? Any default features you need to turn off?

Reply:
Yes, I poked at this a bit.  `toml` didn't have many deps before, but basically it gained a dep on `toml_edit` because they wanted to unify parser and editing logic instead of maintaining two toml implementations. There's no way to disable the edit code, and the edit code doesn't pull in extra dependencies anyway.